src/ProductConstructionService/ProductConstructionService.Api/Api/v2020_02_20/Controllers/CodeFlowController.cs

```diff
@@ -54,4 +54,15 @@ await _workItemProducerFactory
 
         return Ok();
     }
+
+    private class SourceManifestWrapper
```

Comment:
can we just revert this whole file? I don't think we need any of these changes without the VMR page